### ISSUE TITLE
Can't import session

### ISSUE THREAD

**Issue by Wentao-1990** [GMOD/jbrowse-components#2017]
**Describe the bug**

Can't Import session

I saved some sessions using export function, but can't import saved sessions.
**To Reproduce**

**Expected behavior**

**Screenshots**
![Screenshot from 2021-05-26 14-23-10]
(https://user-images.githubusercontent.com/56733533/119733175-006f4f80-be2e-11eb-8d06-4961e8464ece.png)

**Version:**

Firefox
**Additional context**

**Comment by cmdcolin** [comment]
Thanks for catching this, I think I can confirm that this is broken

gives this error in the console


```
Uncaught (in promise) Error: invalid fileLocation
    at openLocation (index.ts:78)
    at onDrop (ImportSessionWidget.js:100)
    at index.js:620
```
This was related to our recent change to allow blob files

**Comment by cmdcolin** [comment]
Possible fix in #2019

**Comment by cmdcolin** [comment]
can perhaps download the latest from that branch with `jbrowse upgrade --branch fix_session_import` to use the build from that branch once it uploads to CI or wait for next release
